Just_Dail_Scrapper.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+import urllib
+import csv
+import constant
+
+
+def scraper():
```

Comment:
Why are you defining these functions inside a function? What purpose does it serve?